packages/editor/src/components/post-text-editor/style.scss

```diff
@@ -1,28 +1,35 @@
-.edit-post-text-editor__body textarea.editor-post-text-editor {
-	border: $border-width solid $light-gray-secondary;
-	display: block;
-	margin: 0;
-	width: 100%;
-	box-shadow: none;
-	resize: none;
+.edit-post-text-editor__body {
```

Comment:
Should the styles for `.edit-post-text-editor__body`  be moved to ` packages/edit-post/src/components/text-editor/style.scss`  instead?

Reply:
(not the textarea itself but its wrapper)